testsuite/MDAnalysisTests/utils/test_log.py

```diff
@@ -81,106 +63,113 @@ def __exit__(self, exc_type, exc_value, traceback):
+    buffer.seek(0)
+    output = "".join(buffer.readlines())
+    _assert_in(output, ('\r' + template).format(**{'step': 1, 'numsteps': n, 'percentage': 100./n}))
+    # last line always ends with \n!
+    _assert_in(output,
+                    ('\r' + template + '\n').format(**{'step': n, 'numsteps': n,
+                                              'percentage': 100.}))
+
+
+def test_custom_ProgressMeter(buffer, n=51, interval=7):
+    template = "RMSD {rmsd:5.2f} at {step:03d}/{numsteps:4d} [{percentage:5.1f}%]"
+    with RedirectedStderr(buffer):
+        pm = MDAnalysis.lib.log.ProgressMeter(n, interval=interval,
+                                              format=template, offset=1)
+        for frame in range(n):
+            rmsd = 0.02 * frame * (n+1)/float(n)  # n+1/n correction for 0-based frame vs 1-based counting
+            pm.echo(frame, rmsd=rmsd)
+    buffer.seek(0)
+    output = "".join(buffer.readlines())
+    _assert_in(output,
+                    ('\r' + template).format(**{'rmsd': 0.0, 'step': 1,
+                                     'numsteps': n, 'percentage': 100./n}))
+    # last line always ends with \n!
+    _assert_in(output,
+                    ('\r' + template + '\n').format(
+                        **{'rmsd': 0.02*n, 'step': n,
+                           'numsteps': n, 'percentage': 100.0}))
+
+
+def test_legacy_ProgressMeter(buffer, n=51, interval=7):
+    template = "RMSD %(rmsd)5.2f at %(step)03d/%(numsteps)4d [%(percentage)5.1f%%]"
+    with RedirectedStderr(buffer):
+        pm = MDAnalysis.lib.log.ProgressMeter(n, interval=interval,
+                                              format=template, offset=1)
+        for frame in range(n):
+            rmsd = 0.02 * frame * (n+1)/float(n)  # n+1/n correction for 0-based frame vs 1-based counting
+            pm.echo(frame, rmsd=rmsd)
+    buffer.seek(0)
+    output = "".join(buffer.readlines())
+    _assert_in(output,
+                    ('\r' + template) % {'rmsd': 0.0, 'step': 1,
+                                       'numsteps': n, 'percentage': 100./n})
+    # last line always ends with \n!
+    _assert_in(output,
+                    ('\r' + template + '\n') % {'rmsd': 0.02*n, 'step': n,
+                                              'numsteps': n, 'percentage': 100.0})
+
+
+@pytest.mark.parametrize('step, percentage', [
+    (1, 100./51),
+    (51, 100.)
+])
+def test_not_dynamic_ProgressMeter(buffer, step, percentage, n=51, interval=10):
+    template = "Step {step:5d}/{numsteps} [{percentage:5.1f}%]"
+    with RedirectedStderr(buffer):
+        pm = MDAnalysis.lib.log.ProgressMeter(n, interval=interval,
+                                              dynamic=False)
+        for frame in range(n):
+            pm.echo(frame)
+    buffer.seek(0)
+    output = "".join(buffer.readlines())
+    _assert_in(output, (template + '\n').format(**{'step': step, 'numsteps': n, 'percentage': percentage}))
+
+
+class TestSetVerbose(object):
```

Comment:
Because there are two methods that basically are testing the same log think. I thought it makes sense to group them. No?